be/src/exprs/fpe.cpp

```diff
@@ -0,0 +1,206 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+#include <string>
+#include <sstream>
+#include <cmath>
+#include <vector>
+#include <ubiq/fpe/ff1.h>
+#include <iomanip>
+
+#include "exprs/fpe.h"
+#include "util/defer_op.h"
+
+namespace starrocks {
+
+const std::string_view FPE::DEFAULT_KEY = "abcdefghijk12345abcdefghijk12345";
+
+std::string_view FPE::trim_leading_zeros(const std::string_view& str) {
```

Comment:
`std::string_view` is better to pass as value not reference. Could see [this](https://stackoverflow.com/questions/60784280/string-view-as-a-parameter-of-const-reference) for detail.

```suggestion
std::string_view FPE::trim_leading_zeros(std::string_view str) {
```